src/js/components/Meter/index.d.ts

```diff
@@ -16,6 +16,7 @@ export interface MeterProps {
   gridArea?: GridAreaType;
   margin?: MarginType;
   max?: number;
+  reverse?: boolean;
```

Comment:
Do we also need to add this to the `propTypes.js` file for Meter?

Reply:
I makes sense adding it, so I added!